### PR TITLE
Update swift3 branch to work with Swift 3, Xcode 8 beta 6

### PR DESCRIPTION
**Pull request by Bersaelor**
@cnoon I saw you were updating the Swift3 branch thus far, I needed it to work with the Beta Seed 6, so I made the necessary changes. Please have a look, the tests and the example app ran fine locally.

### CODE REVIEW

**Comment by ldiqual** [comment]
@Bersaelor Since AlamofireImage depends on Alamofire, you also need to up the Alamofire version to 4 and the iOS deployment version to 9: https://github.com/Alamofire/AlamofireImage/blob/swift3/AlamofireImage.podspec#L22

**Comment by Bersaelor** [comment]
@ldiqual thanks for the reminder, I'm usually using Carthage and forgot the cocoapods.

File: AlamofireImage.podspec [hunk 1]
```diff
@@ -19,10 +19,10 @@ Pod::Spec.new do |s|
     'Source/UIImageView+AlamofireImage.swift'
   ]
 
-  s.ios.deployment_target = '8.0'
-  s.osx.deployment_target = '10.9'
+  s.ios.deployment_target = '9.0'
+  s.osx.deployment_target = '10.11’
```
Comment (ldiqual) [inline AlamofireImage.podspec]: Took me a while to catch it, you have a weird quote here. Should be `'`, not `’`.

**Comment by ldiqual** [comment]
@Bersaelor Thanks! Works great now.

**Comment by cnoon** [comment]
Thanks for putting this together @Bersaelor. I've put together full Beta 6 support in #154. Please check it out when you get a chance. It is a very involved set of changes.

I'm going to go ahead and close this PR out. Please redirect all future comments to #154.

Thanks again. 🍻

**Pull request closed without merging**